src/features/dashboard/DashboardPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Dots from '../../common/components/Icons/Dots'
+import TransactionItem from './TransactionItem'
+import PortfolioColumn from './PortfolioColumn'
+import PortfolioItem from './PortfolioItem'
+import NFTCard from './NFTCard'
+import Notification from './Notification'
+
+const transactions: { type: 'in' | 'out'; amount: string }[] = [
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+]
+
+const notifications = [
+  '1 new listing',
+  '1 new sale',
+  '1 new transfer',
+  '1 new sale',
+  '1 new listing',
+  '1 new sale',
+  '1 new transfer',
+  '1 new sale',
+]
```

Comment:
This data should be coming from the store, consider using `useAppSelector`

Reply:
Sure, only what I'm concerning is that we don't know what data will come from the backend. And it's better not to store the temporary dummy data in the store IMHO.